app/controllers/settings/email_controller.rb

```diff
@@ -19,11 +19,11 @@ def create
       else
         @email_settings_production = @email_settings
       end
-      render :index, status: 422
+      render :index, status: :unprocessable_entity
     end
   end
 
-  private
+private
```

Comment:
i actually like this as it's what i've done in other places i've worked. however it's not something we do here on any of our repos, so maybe we shouldn't start now